app/assets/stylesheets/user_groups/_index.css.scss

```diff
@@ -57,6 +57,20 @@
         }
       }
 
+      .group-invite {
+        @extend .default-group;
+        background: $light-blue;
+        color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.